api_tests/types/bn-chai/index.d.ts

```diff
@@ -1,11 +1,11 @@
-declare module "bn-chai" {
-    function bnChaiFn(ctor: any): Chai.ChaiPlugin;
-
-    export = bnChaiFn;
-}
-
-declare namespace Chai {
-    interface NumberComparer {
-        BN(value: any): void;
-    }
-}
+// declare module "bn-chai" {
```

Comment:
:question: What's going on here?

Reply:
no choice, because we moved from `bn.js` based `ethers.utils.BigNumber` type to `bignumber.js`.